index.html

```diff
@@ -68,8 +69,96 @@ <h1 class="header__page-title">
             </div>
             <div class="main-content">
                 <!--Here we will put our content, erase the filler class and div-->
-                <div class="filler">
-                  
+                <div class="home">
+                    <section class="about-git">
+                        <h3>About GIT</h3>
```

Comment:
make a class for this h3 so it doesnt collide with other h3 in layout.

Reply:
class for h3 has been done